src/Castle.Windsor.Tests/TestUtils.cs

```diff
@@ -12,19 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if !NETCOREAPP1_0 // FirstChanceException event was added in .NET Core 2.0
 namespace CastleTests
 {
 	using System;
 	using System.Collections.Generic;
 	using System.Linq;
 	using System.Runtime.ExceptionServices;
 	using System.Text;
+	using System.Text.RegularExpressions;
 
 	using NUnit.Framework;
 
 	public static class TestUtils
 	{
+#if !NETCOREAPP1_0 // FirstChanceException event was added in .NET Core 2.0
```

Comment:
2.1 is the oldest .NET Core runtime Microsoft still supports. Of course you can decide whether you want to support each version for longer or shorter than Microsoft, but I would recommend adding `netcoreapp2.1` and `netcoreapp3.1` targets to the test project so that the whole suite of tests runs on each supported version of the runtime.

Reply:
We are going through this with Castle Core right now removing support for unsupported .NET Core versions and adding newer .NET Standard 2.x targets, we'll need to do that here once we are done and released there.